[
    {
        "id": 678,
        "name": "Never throw an exception from a Destructor",
        "href": "AIP/quality-rules/678",
        "status": null,
        "critical": true,
        "technologyNames": [
            "C++"
        ]
    },
    {
        "id": 7824,
        "name": "Avoid directly throwing instance of Exception class",
        "href": "AIP/quality-rules/7824",
        "status": null,
        "critical": true,
        "technologyNames": [
            ".NET (Legacy Analyzer)",
            "C#",
            "JEE",
            "VB.NET"
        ]
    },
    {
        "id": 8412,
        "name": "\"CX_ROOT\" should not be used in TRY .. CATCH.. ENDTRY block",
        "href": "AIP/quality-rules/8412",
        "status": null,
        "critical": true,
        "technologyNames": [
            "ABAP"
        ]
    },
    {
        "id": 1020540,
        "name": "Avoid using $http success and error function",
        "href": "AIP/quality-rules/1020540",
        "status": null,
        "critical": false,
        "technologyNames": [
            "HTML5"
        ]
    }
]
